summary_workflows/quantification/README.md

```diff
@@ -0,0 +1,48 @@
+## Matching PA sites to ground truth
+
+The script `match_with_gt.py` uses bedtools window to assign ground truth PAS to the predictions. Each prediction is extended by `n` base pairs, depending on the parameter chosen for `window`.
+
+The script `corr_with_gt.py` calculates the correlation coefficient between prediction and matched ground truth quantification values. The input must be a BED file containing the columns below. Usage: `python3 corr_with_gt.py <prediction_merged.bed>`
+
+Processing steps:
+- run bedtools window for prediction file with itself
+- if necessary, merge prediction sites that fall into the window, expression is summed
+- run bedtools window for merged predictions with ground truth file
+- find multiple ground truth sites overlapping one predicted site
+	- weigths are added for the predicted site based on distance to the ground truth site
```

Comment:
Should be more specific: the weight is the fraction of the predicted site overlapping with the ground truth site (according to the implementation)

Reply:
However, if a predicted site overlaps with multiple ground truth sites, shouldn't rather the predicted site be split?